### PR TITLE
Add nillion, fix arweave URL

### PR DESCRIPTION
**Pull request by hehozo**

### CODE REVIEW

**Comment by suhailkakar** [comment]
thank you, Helen!

i've updated the arweave link, can you please sahre the Nillion's dark version here so I can upload to imagekit?

i've also added you as author for these two eli5s: https://github.com/suhailkakar/web3forall/pull/4

**Comment by hehozo** [comment]
Thanks Suhail, here's the dark version for Nillion!
nillion

**Comment by suhailkakar** [comment]
awesome, thanks!
